docs/source/testing/testing.md

```diff
@@ -3,26 +3,55 @@ title: Integration testing
 description: Utilities for testing Apollo Server
 ---
 
-Testing `apollo-server` can be done in many ways. One simple way is to use ApolloServer's `executeOperation` method to directly execute a GraphQL operation without going through a full HTTP operation.
-
-## `executeOperation`
-
-Integration testing a GraphQL server means testing many things. `apollo-server` has a request pipeline that can support many plugins that can affect the way an operation is executed. The `executeOperation` method provides a single hook to run operations through the request pipeline, enabling the most thorough tests possible without starting up an HTTP server.
+Integration testing a GraphQL server means testing the combination of many interlocking parts at once. One way to test your server setup is to use `ApolloServer`'s `executeOperation` method to directly execute a GraphQL operation without going through a full HTTP operation.
```

Comment:
"full HTTP operation" -> "full HTTP request"

For the sake of disambiguation, I try to use "operation" only when referring to an actual _GraphQL_ operation (query, mutation, subscription), and _not_ when referring to something like an HTTP request that might _contain_ an operation to execute